### DISCUSSION TITLE
Qual versão eu instalo?

### DISCUSSION THREAD

**Discussion by eoliverdev** [balta-io/forum#86] [Tira Dúvidas] [answered]
Olá, estou fazendo Carreira Desenvolvedor Backend .NET > Fundamentos do C#.
Estou na aula .NET "Instalação".
Qual versão do .NET eu instalo? A desatualizado do vídeo ou posso instalar o .NET 6.0?

Grato,
Emerson

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @eoliverdev segue a mesma versão do curso, até por que muitas empresas ainda usam a versão 5 ou anterior.

Depois temos cursos com as novas versões, inclusive hoje, junto com o lançamento da versão 7, temos curso novo já com a versão nova <3
